src/test/scala/shine/DPIA/Store.scala

```diff
@@ -0,0 +1,51 @@
+package shine.DPIA
+
+import rise.core._
+import rise.core.TypedDSL._
+import rise.core.types._
+import util.gen
+
+class Store extends shine.test_util.Tests {
+
+  test("scalar values") {
```

Comment:
looks like a duplicate of another test https://github.com/rise-lang/shine/blob/master/src/test/scala/shine/DPIA/Primitives/Let.scala#L12 ? maybe we should only keep one.

Reply:
I have removed the `Let` test.